biz.aQute.bndlib/src/aQute/bnd/osgi/Verifier.java

```diff
@@ -531,6 +532,22 @@ public void verify() throws Exception {
 		verifyCapabilities();
 		verifyMetaPersistence();
 		verifyPathNames();
+
+		doVerifierPlugins();
+	}
+
+	private void doVerifierPlugins() {
+		for (VerifierPlugin plugin : getPlugins(VerifierPlugin.class)) {
+			try {
+				Processor previous = beginHandleErrors(plugin.toString());
+				plugin.verify(analyzer);
+				endHandleErrors(previous);
```

Comment:
You can fix both in this PR. I would put the fix for the Analyzer into a separate commit.